test/e2e/tests/multi_zone_e2e_test.go

```diff
@@ -257,7 +257,11 @@ func testLifecycleWithVerify(volID string, volName string, instance *remote.Inst
 	if secondMountVerify != nil {
 		// Mount disk somewhere else
 		secondPublishDir := filepath.Join("/tmp/", volName, "secondmount")
```

Comment:
Sorry if I'm missing something from the test setup, but will this test multizonal shared PD?

Reply:
Shared PD is yet not supported on multizone. This is the fix added to handle function testLifecycleWithVerify with useBlock=true.

Reply:
Thanks for clarifying.